webapp/components/create_post.jsx

```diff
@@ -322,10 +322,8 @@ export default class CreatePost extends React.Component {
         document.addEventListener('keydown', this.showShortcuts);
     }
 
-    componentDidUpdate(prevProps, prevState) {
-        if (prevState.channelId !== this.state.channelId) {
-            this.focusTextbox();
-        }
+    componentDidUpdate() {
+        this.focusTextbox();
```

Comment:
It does not on my local platforms. If the textbox is already focused (by typing) focusing it again shouldn't do anything.